src/util/cqlCoverageBuilder/templates/passFailHighlightingTemplate.ts

```diff
@@ -46,3 +48,12 @@ export const clauseNotApplicableStyle = {
   "background-color": "white",
   color: "black",
 };
+
+// blue with dashed underline
+export const clauseHighlightedStyle = {
+  color: "#036CB4",
+  "text-decoration": "underline",
+  "text-decoration-style": "dashed",
+  "text-decoration-color": "#036CB4",
```

Comment:
can we add a bit of a gap between the underline and text. in my local it is lil bit overllaped